test/cloudtest/test_compute.py

```diff
@@ -233,7 +233,9 @@ def test_cluster_replica_sizes(mz: MaterializeApplication) -> None:
         assert replica_id is not None
 
         expected = value.get("selectors", {}) | {"materialize.cloud/disk": "true"}
-        node_selectors = json.loads(get_node_selector(mz, cluster_id, replica_id)[1:-1])
+        node_selectors_raw = get_node_selector(mz, cluster_id, replica_id)
+        print(f"node selectors raw: {node_selectors_raw}")
```

Comment:
Minor: Is this temporary? If not, do we always want to print this or just if it fails?

Reply:
I want to always see it for now just for debugging, temporary though.